deps.bzl

```diff
@@ -672,6 +672,9 @@ def install_go_mod_dependencies(workspace_name = "buildbuddy"):
     )
     go_repository(
         name = "com_github_buildbuddy_io_tensorflow_proto",
+        build_directives = [
+            "gazelle:go_grpc_compilers @io_bazel_rules_go//proto:go_proto,@io_bazel_rules_go//proto:go_grpc_v2",
```

Comment:
note that only `go_grpc_compilers` works here and not `go_proto_compilers`.
Likely because there is a service defined in https://github.com/buildbuddy-io/tensorflow-proto/blob/929b41ab4dc69177bbf84e6562baeaf06f640387/tensorflow_serving/apis/prediction_service.proto#L15